src/main/resources/config.yml

```diff
@@ -2,7 +2,7 @@
 
 
 # Plugin language. Empty: system language.
-# Available: en-US (default, fallback), fr-FR, ru-RU, de-DE, zh-CN, ja-JP.
+# Available: en-US (default, fallback), fr-FR, ru-RU, de-DE, zh-CN, ja-JP, es-US.
```

Comment:
Is there a real difference between es-US and es-ES ?

Reply:
Oh yeah, I missed that. It is es-ES. Sorry.

Reply:
No problem, I can edit it right now if you want me to